cost/aws/idle_compute_instances/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.0
+
+- applying data normalization updates for spend recommendations api
```

Comment:
Do we want to add more specifics to summarize the changes in each of these policies?
For example:
```suggestion
- Normalizing fields for recommendations:
  - Renamed 'tag_set' to 'tags'
  - Renamed 'accountId' to 'accountID'
  - Renamed 'id' to 'resourceID'
  - Renamed 'instanceType' to 'resourceType'
  - Changed 'savings' to be a plain number
  - Added 'savings_currency' to record the currency symbol.
  - Added a 'service' field, hardcoded to "EC2"
```

Reply:
yes, curt actually mentioned being very specific for the 3.0. I also missed updating this to be more specific. again thanks for the catch. I've updated these requests.